lib/core/src/Cardano/Wallet/Primitive/Types.hs

```diff
@@ -1351,11 +1354,17 @@ slotParams gp =
         (gp ^. #getGenesisBlockDate)
         (gp ^. #getActiveSlotCoefficient)
 
+-- | Protocol parameters that can be changed through the update system.
+--
 data ProtocolParameters = ProtocolParameters
     { decentralizationLevel
         :: Quantity "percent" Percentage
+        -- ^ The current level of decentralization in the network.
```

Comment:
You could add that this is the _d_ parameter from the formal spec.

Reply:
Fixed in a417440ceb99cfc997651e6055eefc6453048470.